docs/diagrams/AppointmentListUML.puml

```diff
@@ -0,0 +1,22 @@
+@startuml
+
+class AppointmentList {
+-appointments : Arraylist<appointment> = null
+-returnedFinderArray : Arraylist<appointment> = null
++getAppointment(String) : Appointment
++getList() : Arraylist<appointment>
++add(String[]) : void
++remove(String) : void
++edit(String[]) : void
++view() : void
++view(String) : void
```

Comment:
Should we follow the notation taught in class? e.g. + for public, - for private etc.